ajax/libs/jsignature/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "jsignature",
```

Comment:
Let's update this to `jSignature` which is exactly the same as the repo name, also don't forget to update folder name.